ceph-fs/src/tests/target.py

```diff
@@ -47,27 +47,28 @@ def tearDown(self):
                     logging.warning(
                         "Failed to cleanup mounts on {}".format(unit))
 
-    def _mount_share(self, unit_name: str,
-                     retry: bool = True):
+    def _mount_share(self, unit_name: str, retry: bool = True):
         self._install_dependencies(unit_name)
         self._install_keyring(unit_name)
-        ssh_cmd = (
-            'sudo mkdir -p {0} && '
-            'sudo ceph-fuse {0}'.format(self.mount_dir)
-        )
+        cmd = 'sudo mkdir -p {0} && sudo ceph-fuse {0}'.format(
+            self.mount_dir)
+
         if retry:
-            for attempt in Retrying(
-                    stop=stop_after_attempt(5),
-                    wait=wait_exponential(multiplier=3,
-                                          min=2, max=10)):
-                with attempt:
-                    zaza.utilities.generic.run_via_ssh(
-                        unit_name=unit_name,
-                        cmd=ssh_cmd)
+            @retry(
```

Comment:
This clashes with the local variable above. Maybe the import from tenacity should be renamed.